p4-16/psa/PSA.mdk

```diff
@@ -421,41 +447,40 @@ error to instantiate the `clone` extern in the P4 control or parser
 block.
 
 A PSA implementation provides two configuration metadata, `clone`
-and `clone_spec`, to the PRE to control the cloning mechanism. The
-`clone` enables/disables the cloning mechanism. If the `clone`
-bit is set, the clone mechanism generates a cloned packet with the
-optional attached metadata. If the `clone` bit is unset or
-uninitialized, the clone mechanism is disabled and a cloned packet is
+and `clone_port`, to the PRE to control the cloning mechanism. When
+the `clone` bit is set, the clone mechanism generates a cloned packet with the
```

Comment:
Maybe replace "bit is set" with "bool is true", to match the currently proposed type of 'bool' for the clone field?